### ISSUE TITLE
FNX-4856 ⁃ A11y action for the Quick action bar handle

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#1186]
## Steps to reproduce
1. Turn talkBack on.
1. Open a webpage in Fenix, swipe to reach the quick action bar handle.
    1. Expected behavior
The handle button name is read and on double tap, it expands/collapses the q.a.b.
    1. Actual behavior
The quick action sheet handle is read but it doesn't have the function to collapse/expand the bar.
    1. Device information

* Android device: Pixel (Android 9), Galaxy Tab (Android 8)
* Fenix version: 3/22 build.
